kolibri/core/assets/src/api-resources/task.js

```diff
@@ -1,9 +1,68 @@
 const Resource = require('../api-resource').Resource;
+const logging = require('logging').getLogger(__filename);
 
 class TaskResource extends Resource {
   static resourceName() {
     return 'task';
   }
+
+  localExportContent(driveId) {
+    const promise = new Promise((resolve, reject) => {
+      const clientObj = { path: this.localExportUrl(), entity: { id: driveId },
+        headers: { 'Content-Type': 'application/json' } };
```

Comment:
Hrm, I tell a lie. But, maybe we should wrap the client to always set the headers to application/json, rather than having to pass it in with every API request?